components/empty-state/empty-state-styles.js

```diff
@@ -20,6 +21,12 @@ export const emptyStateStyles = css`
 		display: inline;
 	}
 
+	.d2l-empty-state-description:${unsafeCSS(getFocusPseudoClass())} {
+		border-radius: 0.3rem;
+		outline: 2px solid var(--d2l-color-celestine);
+		outline-offset: 3px;
+	}
```

Comment:
Trying out just using normal outline instead of the `box-shadow` hack. Visually this looks the same (actually a bit) but this will work well in Windows High Contrast Mode.